examples/smoke-co-alarm-app/silabs/src/ZclCallbacks.cpp

```diff
@@ -65,3 +66,184 @@ bool emberAfPluginSmokeCoAlarmSelfTestRequestCommand(EndpointId endpointId)
 {
     return AlarmMgr().StartSelfTesting();
 }
+
+bool emberAfHandleEventTrigger(uint64_t eventTrigger)
+{
+    bool success = false;
+
+    switch (eventTrigger)
+    {
+    case 0xffffffff00000090:
+        success = SmokeCoAlarmServer::Instance().SetSmokeState(1, chip::app::Clusters::SmokeCoAlarm::AlarmStateEnum::kWarning);
+        if (success)
+        {
+            success = SmokeCoAlarmServer::Instance().SetExpressedState(
+                1, chip::app::Clusters::SmokeCoAlarm::ExpressedStateEnum::kSmokeAlarm);
+        }
+        break;
+
+    case 0xffffffff000000a0:
+        success = SmokeCoAlarmServer::Instance().SetSmokeState(1, chip::app::Clusters::SmokeCoAlarm::AlarmStateEnum::kNormal);
+        if (success)
+        {
+            success =
+                SmokeCoAlarmServer::Instance().SetExpressedState(1, chip::app::Clusters::SmokeCoAlarm::ExpressedStateEnum::kNormal);
+        }
+        break;
+
+    case 0xffffffff00000091:
+        success = SmokeCoAlarmServer::Instance().SetCOState(1, chip::app::Clusters::SmokeCoAlarm::AlarmStateEnum::kWarning);
+        if (success)
+        {
+            success = SmokeCoAlarmServer::Instance().SetExpressedState(
+                1, chip::app::Clusters::SmokeCoAlarm::ExpressedStateEnum::kCOAlarm);
+        }
+        break;
+
+    case 0xffffffff000000a1:
+        success = SmokeCoAlarmServer::Instance().SetCOState(1, chip::app::Clusters::SmokeCoAlarm::AlarmStateEnum::kNormal);
+        if (success)
```

Comment:
Do you need checks on priority of expressed state?

Reply:
Recommend creating a method that captures *some* priority on the underlying alarms and sets the `ExpressedState` attribute based on the underlying ongoing alarms